src/index.html

```diff
@@ -4,9 +4,382 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./styles/main.css">
```

Comment:
Sorry I missed this last thing - This should be `scss` file, you don't have to compile it to `css` at all
```suggestion
    <link rel="stylesheet" href="./styles/main.scss">
```